tests/tasks/generators/clusterloader/load-slos.yaml

```diff
@@ -84,11 +84,11 @@ spec:
       CL2_DISABLE_DAEMONSETS: true
       CL2_RATE_LIMIT_POD_CREATION: false
       # enable SLOs related CL2 vars
-      CL2_NETWORK_LATENCY_THRESHOLD: 1s
+      CL2_NETWORK_LATENCY_THRESHOLD: 20s
       CL2_ENABLE_VIOLATIONS_FOR_NETWORK_PROGRAMMING_LATENCIES: true
-      CL2_NETWORK_PROGRAMMING_LATENCY_THRESHOLD: 30s
-      CL2_ENABLE_DNSTESTS: true
-      CL2_USE_ADVANCED_DNSTEST: true
+      CL2_NETWORK_PROGRAMMING_LATENCY_THRESHOLD: 300s
+      CL2_ENABLE_DNSTESTS: false
```

Comment:
Why we are turning this off ? also can you provide some context on how increasing `CL2_NETWORK_LATENCY_THRESHOLD` and `CL2_NETWORK_PROGRAMMING_LATENCY_THRESHOLD` will help us ? does this mean our latency is too high ?